### ISSUE TITLE
Quotes within docstrings result in unparsable schema from _Service.sdl

### ISSUE THREAD

**Issue by fotoetienne** [apollographql/federation-jvm#71]
Example:

```graphql
type Query {
    """Field docstring contains a "quoted" field."""
    fooField: String
}
```

When queried from sdl becomes

```graphql
schema {
  query: Query
}

type Query {
 "Field docstring contains a "quoted" field."
  fooField: String
}
```

The description string should be surrounded in triple quotes as in the original schema.

**Comment by fotoetienne** [comment]
Looks like the problem is [here](https://github.com/apollographql/federation-jvm/blob/master/graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/FederationSdlPrinter.java#L936). For single-line descriptions, we automatically use single-quotes. We should force triple-quotes if there are any characters that can't be contained in a graphql [StringValue](https://spec.graphql.org/draft/#sec-String-Value) (`"`, `\`, or `LineTerminator`)

**Comment by sachindshinde** [comment]
Thanks for reporting this bug! We've merged a fix in, which has been released in v0.4.2.
